src/plan/generational/immix/gc_work.rs

```diff
@@ -9,6 +10,7 @@ impl<VM: VMBinding> crate::scheduler::GCWorkContext for GenImmixNurseryGCWorkCon
     type VM = VM;
     type PlanType = GenImmix<VM>;
     type ProcessEdgesWorkType = GenNurseryProcessEdges<VM, Self::PlanType>;
+    type TPProcessEdges = GenNurseryProcessEdges<VM, Self::PlanType>;
```

Comment:
This should be `UnsupportedProcessEdges`. Genimmix uses a copying nursery which cannot pin objects.